application/views/interface_assets/header.php

```diff
@@ -394,10 +394,12 @@ function handleKeyPress(event) {
 								}
 
 								$oqrs_requests = $this->oqrs_model->oqrs_requests($location_list);
+								if ($oqrs_requests > 0) {
```

Comment:
I'm not happy with this one. If OQRS is enabled we should show the menu entry for it. So the user knows where to find the requests. Also this does not work as you wrote it. 

```suggestion
							
```

Reply:
okay, what's the point of a menu item that shows nothing but a empty page?
and as you can see, it's an optional commit in this PR.

![grafik](https://github.com/wavelog/wavelog/assets/11771323/9b6fa013-e38e-47e8-a43d-e5c51009afc0)

Reply:
We just discussed that in team. 

We don't want to hide this menu entry if OQRS is enabled, even if there are no requests yet. 
You can hide the menu entry if OQRS is disabled.

Reply:
> I am against hiding the menu item. It counts for requests with state < 2, that means menu item will only show when you have new requests. What if you want to check something on old requests?

It shows the menu entry anyway but it shows a small number with new requests if there are more then 2 new

<img width="630" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/29a7fc56-d671-478f-8ddc-f287445a4a5d">

Imho it should stay like this. Just hide it if OQRS is disabled @dg0tm

Reply:
<img width="200" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/69d2d450-8332-47ef-a98c-dc3776d76e85">